### DISCUSSION TITLE
Speed camera / Blitzer

### DISCUSSION THREAD

**Discussion by ToToGSARider** [devemux86/cruiser#257] [Q&A] [answered]
Hello all,

in the last update in the Play Store was indicated as a new feature speed camera. Where can I activate this or what settings must be set? Have nothing found here with the full text search under Speed Camera 

Hallo zusammen,

im letzten Update wurde im Play Store als Neuerung Blitzer angegeben. Wo kann ich das aktivieren bzw. welche Einstellungen muß man setzen? Habe hier mit der Volltextsuche nichts gefunden unter Speed Camera

**Comment by devemux86** [comment, marked as answer]
Hi @ToToGSARider,

It is only 2 steps:
- Download / activate the POI for your region (image 1)
Menu "Downloads | POI"
- Enable the display / voice alerts for the POI (image 2)
"Settings | Navigation | Display | Nearby points of interest"
"Settings | Voice guidance | Announcement | Nearby points of interest"

Es sind nur 2 Schritte:
- Download / Aktivierung der POI für Ihre Region (Bild 1)
Menü "Herunterladen | POI"
- Aktivieren Sie die Anzeige / Sprachausgabe für den POI (Abbildung 2)
"Einstellungen | Navigation | Display | Nahegelegene Orte von Interesse"
"Einstellungen | Sprachausgabe | Ankündigung | Nahegelegene Orte von Interesse"


<img src="https://github.com/devemux86/cruiser/assets/3484020/9cc4482a-158f-416c-bd28-ce4f85bbc3a2" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/d61c4250-ac76-49a5-aecf-e4b3857008ea" width="50%"/>

**Comment by ToToGSARider** [comment]
Okay, is it enough if I only activate the speed cameras, set the number above to 1 or what does this number do? Otherwise I've already found the setting now :-) 

Okay, reicht es, wenn ich nur die Blitzer aktiviere, die Zahl oben auf 1 stelle oder was bewirkt diese Zahl? Ansonsten habe ich die Einstellung jetzt schon gefunden :-)

**Reply by devemux86** [reply]
The number allows to display multiple POI, e.g. the next 2 Speed cameras or Fuel stations.
You can also change the position of the panels (right or left).

Die Nummer ermöglicht es, mehrere POI anzuzeigen, z. B. die nächsten 2 Blitzer oder Tankstellen.
Sie können auch die Position der Tafeln ändern (rechts oder links).

**Comment by ToToGSARider** [comment]
Just tested it in the office, hammer I am thrilled

Habe es gerade getestet im Büro, Hammer ich bin begeistert

**Reply by Leeb64** [reply]
I use "Blitzer de". Costs a one-time fee of 5 euros.
Advantage: It also warns of these mobile speed camera trailers (if they are reported (by the users)). It works in Germany and France(I don't know about other countries). And for 5 euros?
Saved me several times the price.
Just as a tip...

**Comment by ToToGSARider** [comment]
I also have the Blitzer Pro app. You can continue to use the app in parallel, but I think it's great that in the Cruiser app this function was also built in. 


Die Blitzer Pro App habe ich auch. Man kann die App ja parallel weiter nutzen, jedoch finde ich es super, dass in der Cruiser App diese Funktion auch mit eingebaut wurde.
